homeassistant/components/jellyfin/media_source.py

```diff
@@ -363,14 +363,18 @@ def _get_stream_url(self, media_item: dict[str, Any]) -> str:
 
 def _media_mime_type(media_item: dict[str, Any]) -> str:
     """Return the mime type of a media item."""
-    if not media_item[ITEM_KEY_MEDIA_SOURCES]:
+    if ITEM_KEY_MEDIA_SOURCES not in media_item:
```

Comment:
I think it should be:

```suggestion
    if not in media_item.get(ITEM_KEY_MEDIA_SOURCES):
```

Because of the line below we access the first item from it. Not do we only want to ensure it exists at this point, but we also want to make sure it contains something.